legal-api/src/legal_api/resources/v2/business/business_filings/business_filings.py

```diff
@@ -539,13 +539,17 @@ def check_authorization(identifier, filing_json: dict,
         # for incorporationApplication and registration, get legalType from nameRequest
         else:
             legal_type = filing_json['filing'][filing_type]['nameRequest'].get('legalType')
-
+        
+        if not get_permissions_for_action(filing_type):
+            return jsonify({'message': f'Permission Denied - You are not authorized to submit this type of filing for:' + identifier}), \
+                HTTPStatus.FORBIDDEN
+        
         if not authorized(identifier, jwt, action=['edit']):
-            return jsonify({'message': f'You are not authorized to submit filings for {identifier}.'}), \
+            return jsonify({'message': f'Not Authorized - You are not authorized to submit filings for {identifier}.'}), \
                 HTTPStatus.FORBIDDEN
 
         if not is_allowed(business, state, filing_type, legal_type, jwt, filing_sub_type, filing):
-            return jsonify({'message': f'You are not allowed to submit this type of filing for {identifier}.'}), \
+            return jsonify({'message': f'Not Allowed - You are not allowed to submit this type of filing for {identifier}.'}), \
```

Comment:
This relates to bcgov/entity#29982, where I propose changing the message, since it refers to "type of filing", but in fact there are many reasons why this filing might not be allowed right now, but might be allowed some other time.

If Janis or Jacqueline come up with a better message then you could resolve 29982 right now. If not then just leave this as-is.